_people/wenbo.md

```diff
@@ -0,0 +1,60 @@
+---
+# !!!!
+# The NAME of the file should be your LASTNAME (e.g. houben.md)
+# Do NOT delete any field, only fill in the required information
+# !!!! 
+
+# Your full name (Firstname, Lastname)
+name: Wenbo Sun
+
+# The members are displayed in a hierarchical way, so please choose the role (e.g. Full Professor, Assistant Professor etc) 
+# and filter number (e.g. 1, 2, 3) from this list and fill in the role and filter from below:
+
+# (Affiliated) Full Professor - 1
+# Associate Professor - 2
+# Assistant Professor - 3
+# Postdoctoral Researcher - 4
+# PhD Candidate - 5
+# Research Engineer - 6 
+# Guest Researcher - 7
+# Secretary - 8
+role: PhD Candidate
+filter: 5
+
+# same as filter
+theme-filter: 5
+
+# choose your team(s) from the following list: delta, kappa, epsilon, lambda, cel
+team: [delta]
+
+# provide social URLs (if any)
+linkedin: https://www.linkedin.com/in/wenbo-sun-8a355135/
+twitter: 
+github: 
+
+# look for your image here: https://github.com/wis-delft/wis-delft.github.io/tree/master/assets/img/people 
+# if your image doesn't exist, upload it in assets/img/people 
+# write the exact name of the image as it appears in GitHub  
+image: wenbosun.jpg
+
+# the TUD email address
+email: W.Sun-2@tudelft.nl
+
+# Room number (e.g Room - 840 West 4rd floor)
+office: 
+
+# The name of this file with .html extension instead of .md (If the filename is ionescu.md, the "back" field will be ionescu.html)
+back: wenbo.html
```

Comment:
This field value and the name of the file should be your lastname (e.g. sun.md). Please update